src/routing_table/network_tests.rs

```diff
@@ -115,34 +130,44 @@ impl Network {
             }
         }
 
+        let mut iteration = 1;
         while !merge_own_info.is_empty() {
-            let mut merge_other_info = Vec::new();
+            trace!("+++ Merge own info: iteration #{}", iteration);
```

Comment:
I think we have that in some safe_vault tests, too, and it's handy to be able to quickly switch them on by adding one line to the tests. I'd vote for keeping these for now.